tools/wptrunner/wptrunner/browsers/webkitgtk_minibrowser.py

```diff
@@ -23,17 +23,39 @@ def check_args(**kwargs):
     pass
 
 
+def maybe_add_args(required_args, current_args):
+    for required_arg in required_args:
+        # If the arg is in the form of "variable=value", only add it if
+        # no arg with another value for "variable" is already there.
+        if "=" in required_arg:
+            required_arg_variable = "%s=" % required_arg.split("=")[0]
```

Comment:
I guess I'd call this `required_arg_prefix`, or is `variable` an accepted term for the name of an argument?

Reply:
`required_arg_prefix` seems clearer